packages/Header/Header.vue

```diff
@@ -1,14 +1,23 @@
 <template>
-  <div class="header">
-    <div class="container header-box">
-      <div class="title-box" :class="{ w50: !!image }">
-        <h1>{{ title }}</h1>
-        <slot>
-          <p>{{ description }}</p>
+  <div id="header">
+    <div v-if="image || $slots.image" class="container header-box">
+      <div class="title-box-image" half>
+        <slot name="top" />
+        <h1 v-html="title"></h1>
```

Comment:
Any reason why we use `v-html`? `v-html` is to avoid as much as possible, if there is no good reason, better to replace with `v-text` or just `{{ xxx }}`.

Reply:
I wanted to have to possibility to customize the content with specific html. Juts realized that we should use the slot property for this kind of situation. I removed the v-html.